lib/api/controllers/collectionController.js

```diff
@@ -296,6 +296,11 @@ class CollectionController extends NativeController {
       throw kerror.get('api', 'assert', 'invalid_argument', '"all", "stored", "realtime"');
     }
 
+    if (request.input.args.type) {
+      request.addDeprecation('2.10.1',
```

Comment:
You should use the `auto-version` string instead of the version number. (Your feature will probably be released in the 2.10.2)

Reply:
Done! (Didn't know it also worked outside DeprecatedBadge)